packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -93,15 +93,17 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
         uint256 timestamp,
         bytes ancillaryData,
         int256 proposedPrice,
-        uint256 expirationTimestamp
+        uint256 expirationTimestamp,
+        address currency
     );
     event DisputePrice(
         address indexed requester,
         address indexed proposer,
         address indexed disputer,
         bytes32 identifier,
         uint256 timestamp,
-        bytes ancillaryData
+        bytes ancillaryData,
+        int256 proposedPrice
```

Comment:
This event parameter allows the bot to more easily report the proposedPrice for a dispute transaction, which is useful for the bot owner to know.

Reply:
Similar to the above, aren't disputes pretty rare? Would it be too painful to just directly query the request in question if there's a dispute?

Reply:
Disputes are rare, but this is also why I see little harm adding a param to this event

Reply:
That's fair, the gas impact will probably be low.